src/main/java/io/neonbee/config/CorsConfig.java

```diff
@@ -0,0 +1,271 @@
+package io.neonbee.config;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Global CORS configuration.
+ */
+@DataObject(generateConverter = true, publicConverter = false)
+public class CorsConfig {
+    private boolean enabled;
+
+    private List<String> origins;
+
+    private List<String> relativeOrigins;
```

Comment:
What is the behaviour if we add the same origin/relativeOrigin multiple times? Maybe a Set makes more sense?

Reply:
Awesome, thank you for clarification.